pkg/cluster/oidc.go

```diff
@@ -18,7 +20,7 @@ func oidcEntry() *ConfigManagerEntry {
 			func(c *Config) error {
 				for _, o := range c.OIDCConfigs {
 					if err := o.Validate(); err != nil {
-						return err
+						return apierrors.NewInvalid(anywherev1.GroupVersion.WithKind(anywherev1.OIDCConfigKind).GroupKind(), o.Name, err)
```

Comment:
I know this code was already here, but could we add a test for this case? Since we are changing the logic to return an error.